Tourist-Places/MrRobot-Hyderabad.md

```diff
@@ -0,0 +1,11 @@
+# Tourist Places
+
+- Charminar
+- Tank bund
+- IMAX
+- Golconda Port
+
+# Food
+
+- Chicken Biryani
```

Comment:
Was just in Hyderabad last month. Chicken Biryani was a favorite dish at the hotel :Chicken: